### PR TITLE
MOSIP-45201:PMS-New endpoint PATCH /bio-extractor-configurations/{bio…

### PR DESCRIPTION
**Pull request by damodarguru**
MOSIP-45201: [API Automation] PMS-New endpoint PATCH /bio-extractor-configurations/{bioExtractorConfigurationId}

## Summary by CodeRabbit

* **Tests**
  * Expanded test coverage for bio extractor configuration management, including deletion scenarios.
  * Added comprehensive negative test cases and edge case validations for configuration operations.
  * Enhanced test case dependency chains to include prerequisite validations for policy, authentication, and credential flows.
  * Improved test infrastructure for partner credential mapping and certificate management workflows.

### CODE REVIEW

File: api-test/src/main/resources/config/testCaseInterDependency.json [hunk 1]
```diff
@@ -902,6 +1120,7 @@
         "TC_PMS_RequestAPIKeyForAuthPartner_05",
         "TC_PMS_publishPolicy_01"
     ],
+    "TC_PMS_uploadcaCertMisp_01": ["TC_PMS_createPartnerSelfRegistration_misp__01"],
```
[the remark this answers is not in the document]
Comment (damodarguru) [reply api-test/src/main/resources/config/testCaseInterDependency.json]: Removing mistakenly added

**Review by jayesh12234 — APPROVED** [review]

**Review by mohanachandran-s — APPROVED** [review]

File: api-test/src/main/resources/pms/BioExtractorConfiguration/DeleteBioExtractorConfiguration/DeleteBioExtractorConfiguration.yml [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+      "version": "1.0",
+      "requestTime": "$TIMESTAMP$",
+      "bioExtractorConfigurationId": "$ID:CreateBioExtractorConfiguration_For_Delete_Smoke_sid_id$",
+      "status": "DELETED"
+}'
+    output: '{
+      "status": "Bio Extractor configuration deleted successfully."
+}'
+
+  Pms_DeleteBioExtractorConfiguration_With_Invalid_Id_Neg:
```
Comment (prathmeshj12) [inline api-test/src/main/resources/pms/BioExtractorConfiguration/DeleteBioExtractorConfiguration/DeleteBioExtractorConfiguration.yml]: Keep the Negative test cases in different folder do not add in Positive flow

Comment (damodarguru) [reply api-test/src/main/resources/pms/BioExtractorConfiguration/DeleteBioExtractorConfiguration/DeleteBioExtractorConfiguration.yml]: Updated

**Review by prathmeshj12 — APPROVED** [review]

**Review by mayuradesh — APPROVED** [review]
